### PR TITLE
Adding dynamic_array.cpp

### PR DESCRIPTION
**Pull request by Eunoia1729**
Fixed issue #28

### CODE REVIEW

File: Data Structures/Array/dynamic_array.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/*
+   DYNAMIC ARRAYS : VECTOR (STL)
+*/
+#include<bits/stdc++.h>
+using namespace std;
+void display_array( vector<int> v)
```
Comment (aditya81070) [inline Data Structures/Array/dynamic_array.cpp]: Follow [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#naming-convention).

File: Data Structures/Array/dynamic_array.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    vector<int> :: iterator it;
+    for( it = v.begin() ; it!= v.end() ; ++it)
+        cout<<*it<<" ";
+
+    cout<<endl;
+}
+
+int main()
+{     int i;
+      vector<int> array = {1,2,3,4};   //The syntax for intializing a dynamic array using vector(STL)
```
Comment (aditya81070) [inline Data Structures/Array/dynamic_array.cpp]: Prompt user for input the array.

File: Data Structures/Array/dynamic_array.cpp [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      
+      //To completely clear the array
+      array.clear();
+      
+      //To check if the array is empty
+      if( array.empty() == true)
+        cout<<"Yes, the array is empty\n";
+      else
+        cout<<"No, the array is not empty\n";
+
```
Comment (aditya81070) [inline Data Structures/Array/dynamic_array.cpp]: Remove blank lines.

**Comment by Eunoia1729** [comment]
Is it ok now? @aditya81070

File: Data Structures/Array/dynamic_array.cpp [hunk 4]
```diff
@@ -0,0 +1,86 @@
+//Name: Kumar Raju.B
+//E-mail: bim2017008@iiita.ac.in
+//TITLE: DYNAMIC ARRAY-VECTOR(STL)
+#include<bits/stdc++.h>
+using namespace std;
+void display_array( vector<int> v)
```
Comment (aditya81070) [inline Data Structures/Array/dynamic_array.cpp]: Follow [Coding convention for function name](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#naming-convention)

**Comment by aditya81070** [comment]
@Eunoia1729 Follow coding standard in this code and also in your future codes.

**Comment by Eunoia1729** [comment]
I have commited the request changes now. Kindly review

File: Data Structures/Array/dynamic_array.cpp [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      // Syntax to append a value at the end of the array ,time complexity:O(1)
+      array.push_back(10); 
+      
+      array[3] = 7; // Syntax to modify a value of the array
+      
+      // To find the number of elements in the array ,time complexity:O(1)
+      cout<<"The size of the array is : "<<array.size()<<endl; 
+      
+      //The format to know the value at particular index is :(for ex. at 4) ,time complexity:O(1)
+      cout<<"The value at index 4 is "<<array[4]<<endl;
```
Comment (aditya81070) [inline Data Structures/Array/dynamic_array.cpp]: What if the user enters array size less than 5? This will be a garbage value. Fix it.

**Comment by ramitsawhney27** [comment]
@Eunoia1729 excellent! This is a great thing to learn and it'll help you as well. Ideally, you could add this, and we can approve it then.

**Comment by Eunoia1729** [comment]
@ramitsawhney27 ive added a function to delete even valued elements. Learnt a new thing :100:

**Pull request merged**